Sming/rboot/eagle.rom.addr.v6.ld

```diff
@@ -0,0 +1,347 @@
+PROVIDE ( Cache_Read_Disable = 0x400047f0 );
```

Comment:
Yeah, fixed. Add $(SMING_HOME)/rboot $(SMING_HOME)/rboot/appcode to the end of EXTRA_INCDIR in the Makefile-project.mk file. As you've tested the change to rBoot ld files on Widnows (which was my only real concern) I'll push that change now (and this little one).